pymc3/distributions/bart.py

```diff
@@ -63,22 +63,30 @@ def __init__(
             )
         self.m = m
         self.alpha = alpha
-        self.y_std = Y.std()
-
-        if scale is None:
-            self.leaf_scale = NormalSampler(sigma=None)
-        elif isinstance(scale, (int, float)):
-            self.leaf_scale = NormalSampler(sigma=Y.std() / self.m ** scale)
 
         if inv_link is None:
-            self.inv_link = lambda x: x
+            self.inv_link = self.link = lambda x: x
+            self.init_mean = self.Y.mean()
+            self.Y_un = self.Y
+        elif isinstance(inv_link, str):
+            if inv_link == "logistic":
+                self.inv_link = expit
+                self.link = lambda x: (x - 0.5) * 10
+            elif inv_link == "exp":
+                self.inv_link = np.exp
+                self.link = np.log
+                self.Y[self.Y == 0] += 0.0001
```

Comment:
Should this use np.finfo.eps or something of the sort?

Reply:
I will try. Although to avoid exactly zero value should be OK, np.finfo.eps maybe be too extreme.